app/budget/budget.css

```diff
@@ -0,0 +1,374 @@
+.btn-default {
```

Comment:
I'd recommend namespacing all the css rules in here with .budget-view (after adding that to the root div in your template). Right now this is the only page in the budget module, but more pages will likely be made down the road and we don't want uncontrolled CSS bleeding.